scripts/storage-emulator-integration/conformance/firebase.endpoints.test.ts

```diff
@@ -173,6 +173,40 @@ describe("Firebase Storage endpoint conformance tests", () => {
           })
           .expect(403);
       });
+
+      it("should return 400 when calling finalize on an already finalized resumable upload", async () => {
+        const uploadURL = await supertest(firebaseHost)
+          .post(
+            `/v0/b/${storageBucket}/o/test_upload.jpg?uploadType=resumable&name=test_upload.jpg`
+          )
+          .set({
+            Authorization: "Bearer owner",
```

Comment:
I tested against prod as well, and I notice that prod returns a 200 response. In prod this makes sense because there was an initial rules check and a second finalize is probably seen as a no op. In emulator however we aren't doing the initial rules check and this error is indication of a failed check in the android sdk so I'm not sure if I should match prod behavior or return a different code here.

In any case the emulator stability bug will be fixed but the error message from the android sdk won't make much sense if I return 200 response. What are your thoughts here?